src/Confluent.Kafka/Timestamp.cs

```diff
@@ -36,16 +36,38 @@ public struct Timestamp
         /// <summary>
         ///     Initializes a new instance of the Timestamp structure.
         /// </summary>
+        /// <param name="unixTimestampMs">
+        ///     The unix millisecond timestamp.
+        /// </param>
+        /// <param name="type">
+        ///     The type of the timestamp.
+        /// </param>
+        public Timestamp(long unixTimestampMs, TimestampType type)
+        {
+            Type = type;
+            UnixTimestampMs = unixTimestampMs;
+        }
+
+        /// <summary>
+        ///     Initializes a new instance of the Timestamp structure.
+        ///     Prefer the (long, TimestampType) overload if possible.
```

Comment:
That was because the DateTime in constructor will be likely different to the DateTime property (Utc conversion or rounding) Even with the remarks, it doesn't seem right to me that constructor use a different object than the one given in parameter.

I even wondered about deprecating this method (which we never call with DateTime now, Timestamp class should represent a librdkafka Timestamp, which is implemented from unix ms timestamp) Didn't remove it to not break api (yes, sometime I care about this ;) )